gitbook/result/others.md

```diff
@@ -26,6 +26,13 @@ Converts an Option to a Result, using the given error if Some.
 ```fsharp
 'a -> 'b option -> Result<unit, 'a>
 ```
+### requireNotNull
+
+Converts a nullable value to a Result, using the given error if null.
+```fsharp
+'a -> 'b : null -> Result<'b, 'a>
```

Comment:
I'm not sure what the correct annotation syntax is here for the generic constraint `null`. Do you know?

Reply:
IMHO it's not necessary to specify constraints like that in the docs.